packages/testing-library/utils/browser-control.ts

```diff
@@ -11,10 +16,9 @@ export async function startBrowser(url: string, headless: boolean) {
 	// of each developer and thus leads to inconsistent test results.
 	await page.emulateMedia({ colorScheme: 'light' });
 
-	return { browser, page };
+	return { page };
```

Comment:
Shouldn't we change the return value to just be `page` in that case? An object with `page` seems redundant.

Reply:
nice catch! fixed